CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.22.2] - 2023-06-08 (NOT DEPLOYED)
```

Comment:
I think we should deploy as well after merging this PR as this contains fix. @syedgulahmed